src/nft/components/details/detailsV2/TraitRow.tsx

```diff
@@ -0,0 +1,47 @@
+import Column from 'components/Column'
+import Row from 'components/Row'
+import { Trait } from 'nft/types'
+import { formatEth } from 'nft/utils'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { RarityGraph } from './RarityGraph'
+
+const SubheaderTiny = styled.div`
+  font-size: 10px;
+  line-height: 16px;
+  font-weight: 600;
+  color: ${({ theme }) => theme.textSecondary};
+`
+
+const TraitValue = styled(Column)`
+  gap: 4px;
+  flex: 3;
+`
+
+const TraitRowValue = styled(ThemedText.BodySmall)<{ $flex?: number; justify?: string }>`
+  display: flex;
+  line-height: 20px;
+  padding-top: 20px;
+  flex: ${({ $flex }) => $flex ?? 1};
+  justify-content: ${({ justify }) => justify};
```

Comment:
nit: I wonder whether `$justifyContent` prop wouldn't be more appropriate, just to avoid confusion (I see we're differentiating between configurable CSS properties and the rest).